src/main/webapp/script.js

```diff
@@ -80,7 +87,18 @@ navLogo.addEventListener('click', hideMobileMenu)
 
 // To put the uploaded images back in the webpage
 async function image_upload() {
-  const uploaded_images = await fetch('/list-images');
+  //const uploaded_images = await fetch('/list-images');
+
+
+  const uploaded_images = await fetch('/list-images' , {
+    method: 'POST', // or 'PUT'
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(value),
+  });
+
```

Comment:
Is this the Json Buffer? If so, looks good  👍

Reply:
Yeah I think thats it when we merge and test we will see if it actually works, or I might be mistaken.